phpserver/router.php

```diff
@@ -39,13 +39,23 @@
 
 if (php_sapi_name() === 'cli-server') {
     $debug("URI: {$_SERVER["REQUEST_URI"]}");
+    if($_SERVER["REQUEST_URI"] == '/'|| strpos($_SERVER["REQUEST_URI"],'/admin') !==false){
```

Comment:
We can't hardcode `/admin` path because it can be different according to store settings

Reply:
Though we could pull directly from the config like below. 
```
$admin = isset($etcConfig['backend']['frontName']) ? $etcConfig['backend']['frontName'] : 'NO-CONFIG';

 if($_SERVER["REQUEST_URI"] == '/'|| strpos($_SERVER["REQUEST_URI"],$admin) !==false)
 {
    return false;
}
```